starter-code/js/app.js

```diff
@@ -1,6 +1,66 @@
 /* setup your angular app here */
-
+var app = angular.module('FoodSort', []);
 //debug stuff to show the app is loading and fruit / veggies are available
 console.log('App Started');
 console.log('Fruit count', fruit.length);
-console.log('Veggie count', vegetables.length);
\ No newline at end of file
+console.log('Veggie count', vegetables.length);
+
+app.controller('FoodCtrl', ['$scope', function($scope) {
+
+  $scope.fruit = [
+  'Apple',
+  'Apricot',
+  'Banana'
+  ];
+
+  $scope.vegetables = [
+  'Artichoke',
+  'Arugula',
+  'Asparagus'
+  ];
+
+  $scope.pool = $scope.fruit.concat($scope.vegetables);  
+
+  $scope.fruit= [];
+
+  $scope.vegetables = [];
+
+    // $scope.fruit.concat($scope.vegetables);
+
+  $scope.moveLeft = function(idx) {
+    if ($scope.fruit.indexOf(this.innerHTML) != -1) {
+      return;
+    } else if ($scope.pool.indexOf(this.innerHTML) != -1) {
+        $scope.fruit.push($scope.pool.splice($index));
+      }
+    } else {
+      $scope.fruit.push($scope.vegetables.splice($index));
+    }
+  }
+
+  $scope.moveRight = function(idx) {
+    if ($scope.vegetables.indexOf(this.innerHTML) != -1) {
+      return;
+    } else if ($scope.pool.indexOf(this.innerHTML) != -1) {
+        $scope.vegetables.push($scope.pool.splice($index));
+      }
```

Comment:
Also delete this line here (line 46) which is causing another syntax error in your app.